gymnasium/core.py

```diff
@@ -393,28 +414,30 @@ class RewardWrapper(Wrapper):
     because it is intrinsic), we want to clip the reward to a range to gain some numerical stability.
     To do that, we could, for instance, implement the following wrapper::
 
-        class ClipReward(gym.RewardWrapper):
+        class ClipReward(gymnasium.RewardWrapper):
```

Comment:
Imo it's better to keep it as `gym.RewardWrapper` since that's the recommended usage, and the code snippet isn't (a) related to the gym-gymnasium compatibility, or (b) runnable by itself

Reply:
I will change all of the example code to be consistent